### PR TITLE
Feature/#10 logic player human

### LINKED ISSUE #10: Logic | Player - Human
- [ ]  Funkcja zwracająca obiekt reprezentujący gracza - człowieka.
- [ ]  Obiekt gracza pozwala zadać graczowi pytanie oraz udzielić odpowiedzi na zadane pytanie.
- [ ]  Obiekt po zadaniu i udzieleniu pytania wykonuje funkcje (callbacki) informujące o zadaniu pytania / udzieleniu odpowiedzi.
- [ ]  Działanie komponentu pokryte testami jednostkowymi.

### PR DESCRIPTION
**Pull request by DominikNowak**
closes #10

### CODE REVIEW

File: src/app/logic/PlayerHuman.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class PlayerHuman {
```
Comment (vieraboschkova) [inline src/app/logic/PlayerHuman.js:1]: Note for next task: Perhaps we can create a class Player with methods, and then extend it by creating HumanPlayer and ComputerPlayer

**Review by vieraboschkova — APPROVED** [review]

**Review by synowa — APPROVED** [review]
Ok

File: src/app/logic/PlayerHuman.js (hunk 1 above)
Comment (synowa) [inline src/app/logic/PlayerHuman.js:1]: Suggestion: change the name for "HumanPlayer" for better readability.

**Review by synowa — APPROVED** [review]
Looks good to me.

**Review by Enessetere — APPROVED** [review]
Looks good to me.

**Pull request merged**